x-pack/plugins/cases/public/components/status/translations.ts

```diff
@@ -9,7 +9,7 @@ import { i18n } from '@kbn/i18n';
 export * from '../../common/translations';
 
 export const ALL = i18n.translate('xpack.cases.status.all', {
-  defaultMessage: 'All',
+  defaultMessage: 'All status',
```

Comment:
Copy change required to clarify the "All" in the status filter dropdown now that the severity field was added.